common/app/rendering/core/JavascriptRendering.scala

```diff
@@ -95,11 +96,9 @@ trait JavascriptRendering extends Logging {
     new ByteArrayInputStream(pre.getBytes(StandardCharsets.UTF_8))
   }
 
-  private def loadFile(fileName: String): Try[InputStream] = {
-    Option(getClass.getClassLoader.getResourceAsStream(fileName)) match {
-      case Some(stream) => Success(stream)
-      case None => Failure(new FileNotFoundException(s"${this.getClass.getSimpleName}: Cannot find file '$fileName'. Have you run `make ui-compile`?"))
-    }
+  private def loadFile(file: String): Try[InputStream] = {
+    Try(Paths.get(file)).flatMap(path => Try(Files.newInputStream(path)))
```

Comment:
Any reason not to wrap the whole thing in one `Try`? 

`Try(Files.newInputStream(Paths.get(file)))`